### PR TITLE
Readd TL and remove (penalty: 0.0) from /f who/show

### PR DESCRIPTION
**Pull request by Tibowl**

### CODE REVIEW

File: src/main/java/com/massivecraft/factions/cmd/CmdShow.java [hunk 1]
```diff
@@ -69,18 +76,18 @@ public void perform() {
...
-                msg("<a>Bank contains: <i>" + Econ.moneyString(Econ.getBalance(faction.getAccountId())));
+                msg(TL.COMMAND_SHOW_BANKCONTAINS, Econ.moneyString(Econ.getBalance(faction.getAccountId())));
             }
 
         }
 
-        String allyList = p.txt.parse("<a>Allies: ");
-        String enemyList = p.txt.parse("<a>Enemies: ");
+        String allyList = p.txt.parse("<a>" + TL.COMMAND_SHOW_ALLIES);
+        String enemyList = p.txt.parse("<a>" + TL.COMMAND_SHOW_ENEMIES);
```
Comment (drtshock) [inline src/main/java/com/massivecraft/factions/cmd/CmdShow.java:90]: We wouldn't need to have the text parsing as well as the <a> if we are using TL.

Comment (Tibowl) [inline src/main/java/com/massivecraft/factions/cmd/CmdShow.java:90]: Its for the <a> color (previously .color gold) currently on mobile and I
don't have access to my pc, gonna do it tomorrow if you haven't merged &
edited it yourself.
Op 3-jan.-2015 23:30 schreef "Trent Hensler" notifications@github.com:

File: src/main/java/com/massivecraft/factions/cmd/CmdShow.java [hunk 2]
```diff
@@ -104,8 +111,8 @@ public void perform() {
         sendMessage(allyList);
         sendMessage(enemyList);
 
-        String onlineList = p.txt.parse("<a>") + "Members online: ";
-        String offlineList = p.txt.parse("<a>") + "Members offline: ";
+        String onlineList = TL.COMMAND_SHOW_MEMBERSONLINE;
+        String offlineList = TL.COMMAND_SHOW_MEMBERSOFFLINE;
```
Comment (drtshock) [inline src/main/java/com/massivecraft/factions/cmd/CmdShow.java:115]: This doesn't compile.

**Comment by drtshock** [comment]
You should use your IDE when making PRs.

**Pull request closed without merging**